html-web-page/cypress.config.mjs

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
```

Comment:
I can definitely expose `VITE_APP_PORT` and we can use that wherever needed, vite just typically exposes it based on a config, which is what i hardcoded to match here.

Reply:
The thing is I am running an app on my laptop (it is up all the time) on port `3000`. I was thinking to change its port, but it is not that pressing. :man_shrugging:

Still, I think there should be a default value. I am not sure how can we do it on Windows/PowerShell, but in Bash, it can be done using `${ROMCAL_APP_PORT:-3000}` (i.e. use the env var value, but if it is empty/undefined / an empty string, use `3000`).

Reply:
oh, certainly, that's definitely doable, in general